chainntnfs/txnotifier.go

```diff
@@ -322,6 +335,20 @@ func NewSpendRequest(op *wire.OutPoint, pkScript []byte) (SpendRequest, error) {
 	}
 	r.PkScript = outputScript
 
+	// For Taproot spends we have the main problem that for the key spend
+	// path we cannot derive the pkScript from only looking at the input's
+	// witness. So we need to rely on the outpoint information alone.
```

Comment:
We should add a TODO here to eventually be able to detect classes of scriptpath spends, since we can re-compute the previous output script from the control block.

Reply:
Good idea. Added the TODO.